tests/ember_debug/object-inspector-test.js

```diff
@@ -712,7 +712,7 @@ module('Ember Debug - Object Inspector', function(hooks) {
 
     let message = await inspectObject(proxy);
 
-    assert.ok(message.name.includes('(unknown)'));
+    assert.ok(message.name.includes('ObjectProxy'), 'object name should start with <ObjectProxy:');
```

Comment:
@patricklx Is this the intended output? I suppose it is better than `unknown`, but just want to make sure things are working as intended.

Reply:
yes, since getClassName replaces that part with the class name. But i think it only does it dev builds. in prod it will show unknown again

Reply:
Is there not a way to show the same things in both?

Reply:
no, since the the class name comes from constructor.name, which will get uglified in production